lib/runners/schema/options.rb

```diff
@@ -8,7 +8,7 @@ module Schema
         base: string?,
         git_url: string,
         git_url_userinfo: string?,
-        pull_number: number?,
+        refspecs: array?(string),
```

Comment:
[suggestion] How about using the name `refspec` instead of `refspecs` and allowing one refspec as a string value?

```suggestion
        refspec: enum?(string, array(string)),
```

Reply:
I will change it as you said.

Note I deliberately defined it not using `enum?` because I want to make it obvious from the parameter name. I thought this parameter is basically called by developers, not end-users, so clarity was important.